### DISCUSSION TITLE
How to pass the dialect in configuration file?

### DISCUSSION THREAD

**Discussion by jarimustonen** [sql-formatter-org/sql-formatter#765] [Q&A]
How can I pass the dialect (like `postgresql`) in the configuration file (`.prettierrc`) or as a command line option?

**Comment by nene** [comment]
I suppose you're using this library through a prettier plugin. Perhaps [prettier-plugin-sql](https://github.com/un-ts/prettier/tree/master/packages/sql).

Perhaps ask the authors of the plugin. Or look for general documentation regarding passing options to prettier plugins.

**Comment by jarimustonen** [comment]
You are correct. Sorry to post this on the wrong board.
